test/Microsoft.Health.Fhir.Shared.Tests.Crucible/CrucibleTestFixture.cs

```diff
@@ -37,12 +40,14 @@ public void Run(string id)
 
             if (findTest != null)
             {
+                bool isCosmosDb = CrucibleDataSource.GetDataStore().Equals(DataStore.CosmosDb);
                 var failures = findTest.Result
                     .Where(x =>
                     {
                         var testName = $"{x.TestId ?? findTest.TestId}/{x.Id}";
-                        return x.Status == "fail" && !KnownCrucibleTests.KnownFailures.Contains(testName) && !KnownCrucibleTests.KnownBroken.Contains(testName)
+                        bool commonFailures = x.Status == "fail" && !KnownCrucibleTests.KnownCommonFailures.Contains(testName) && !KnownCrucibleTests.KnownBroken.Contains(testName)
                                && !x.Message.ToString().Contains(KnownCrucibleTests.BundleCountFilter);
+                        return isCosmosDb ? commonFailures && !KnownCrucibleTests.KnownCosmosDbFailures.Contains(testName) : commonFailures && !KnownCrucibleTests.KnownSqlServerFailures.Contains(testName);
```

Comment:
As a suggestion we might be able to simplify by creating a `knownFailures` variable with this logic built-in to use here and below.
e.g. `var knownFailures = commonFailures.Concat(isCosmosDb ? KnownCrucibleTests.KnownCosmosDbFailures : KnownCrucibleTests.KnownSqlServerFailures).ToArray();`